### PR TITLE
enforce that AWS auth goes through the correct .aws/credentials profile

### PR DESCRIPTION
**Pull request by samsonasu**

### CODE REVIEW

**Review by tdipiazza — APPROVED** [review]
:shipit:

File: CHANGELOG.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ This project attempts to follow [semantic versioning](https://semver.org/).
     * No more group_vars directory
     * All of the host/group configuration is in that one file now!
   * BREAKING: `subspace vars` is now `subspace secrets`
+  * BREAKING: sidekiq_concurrency renamed to sidekiq_workers, default changed from 10 -> 1
```
Comment (Nonsense331) [inline CHANGELOG.md:23]: can you get rid of the default instead?

**Comment by Nonsense331** [comment]
nice! 🛳️

File: CHANGELOG.md (hunk 1 above)
Comment (samsonasu) [reply CHANGELOG.md:23]: I left it at one which is safe but I see your point

**Pull request merged**